test/test262.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+cd "$(dirname "$0")"
+if [ ! -d "tc39" ]; then
+  git clone --depth=1 https://github.com/tc39/proposal-temporal.git tc39
+else
+  cd ./tc39
+  git fetch origin
+  git merge --ff-only origin/main
+fi
+
+cd ../ && TEST262=1 npm run build
+cd tc39/polyfill && npm install
+PRELUDE=../../../dist/script.js npm run test262
```

Comment:
To support debugging in #46 (or just CLI running a subset of tests), I'd suggest this change: 

```suggestion
PRELUDE=../../../dist/script.js npm run test262 "$TESTS"
```

Another approach would be to accept a command-line parameter, but the problem with that approach is we'd need to transform the directory because you can't use the same glob pattern for this test262.sh that you can use for the nested TC39 version. You'd need to strip off the extra relative path.

Reply:
The env variable seems reasonable to me - let's go with that. If we don't like it we can just change it later. Consistency between the two repos for running tests seems rather important IMO.